### PR TITLE
ENG-4744 Added MDC filters for tenant and user

### PR DESCRIPTION
**Pull request by zonia3000**

### CODE REVIEW

File: engine/src/main/java/org/entando/entando/web/filter/MDCTenantFilter.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import org.slf4j.MDC;
+
+public class MDCTenantFilter extends HttpFilter {
+
+    private static final String MDC_KEY_TENANT = "tenant";
+
+    @Override
+    protected void doFilter(HttpServletRequest request, HttpServletResponse response, FilterChain chain)
+            throws IOException, ServletException {
+        MDC.put(MDC_KEY_TENANT, ApsTenantApplicationUtils.getTenant().orElse(ITenantManager.PRIMARY_CODE));
```
Comment (entgigi) [inline engine/src/main/java/org/entando/entando/web/filter/MDCTenantFilter.java]: you should use a try catch finally to remove always from MDC

File: engine/src/main/java/org/entando/entando/web/filter/MDCUserFilter.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import org.slf4j.MDC;
+
+public class MDCUserFilter extends HttpFilter {
+
+    private static final String MDC_KEY_USER = "user";
+
+    @Override
+    protected void doFilter(HttpServletRequest request, HttpServletResponse response, FilterChain chain)
+            throws IOException, ServletException {
+        MDC.put(MDC_KEY_USER, getCurrentUser(request));
```
Comment (entgigi) [inline engine/src/main/java/org/entando/entando/web/filter/MDCUserFilter.java]: you should use a try catch finally to remove always from MDC

**Review by entgigi — CHANGES_REQUESTED** [review]

File: engine/src/main/java/org/entando/entando/web/filter/MDCTenantFilter.java (hunk 1 above)
Comment (zonia3000) [reply engine/src/main/java/org/entando/entando/web/filter/MDCTenantFilter.java]: done

File: engine/src/main/java/org/entando/entando/web/filter/MDCUserFilter.java (hunk 2 above)
Comment (zonia3000) [reply engine/src/main/java/org/entando/entando/web/filter/MDCUserFilter.java]: done

**Review by entgigi — APPROVED** [review]
LGTM

**Pull request merged**